sdcm/cluster.py

```diff
@@ -2450,28 +2458,28 @@ def get_seed_nodes_from_node(self):
                 try:
                     self.seed_nodes_ips = conf_dict['seed_provider'][0]['parameters'][0]['seeds'].split(',')
                 except Exception as details:
-                    self.log.debug('Loaded YAML data structure: %s', conf_dict)
-                    self.log.error('Scylla YAML config contents:')
+                    self.log.debug('Loaded YAML data structure: %s', conf_dict)  # pylint: disable=no-member
+                    self.log.error('Scylla YAML config contents:')  # pylint: disable=no-member
```

Comment:
do we need it here?

Reply:
Yes cause logger is set in the sibling class. Hence linter doesn't recognize it.
Using the global logger we'll loose the context, i.e. from which node this print case from.